client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.scss

```diff
@@ -52,6 +52,10 @@
 			height: 28px;
 			padding: 2px;
 
+			&:last-child:not(.has-icon) {
+				padding-right: 6px;
```

Comment:
This looks a bit odd when there's only the "Remove" button: 
![Screenshot 2023-03-24 at 2 45 51 PM](https://user-images.githubusercontent.com/797888/227446505-a541be0e-c593-4175-9cc1-53e6c950ccab.png)

I assume that the `padding-right: 6px` is to match the space for the move up/down arrows? Otherwise the code looks good!

Reply:
Ya, it does look a bit odd when it just "remove".  See what @davewhitley thinks, it is something that is easy to change so not a big worry.

Reply:
Fixed!

![image](https://user-images.githubusercontent.com/13596067/227448849-f74f5a9e-06db-42fa-b575-83be74d040ce.png)